src/lib.rs

```diff
@@ -45,4 +51,6 @@ pub mod prelude {
     pub use algorithm::from_postgis::FromPostgis;
     #[cfg(feature = "postgis-integration")]
     pub use algorithm::to_postgis::ToPostgis;
+    #[cfg(feature = "projection")]
```

Comment:
Maybe I missed it, but I don't see `projection` as a defined feature--should this be `feature = "proj"`?

Reply:
It absolutely should. Fixed!